src/Sets/SimpleSparsePolynomialZonotope.jl

```diff
@@ -0,0 +1,105 @@
+"""
+    SimpleSparsePolynomialZonotope{N, VN<:AbstractVector{N}, MN<:AbstractMatrix{N}, ME<:AbstractMatrix{<:Integer}} <: AbstractPolynomialZonotope{N}
+
+Type that represents a sparse polynomial zonotope.
+
+Given a constant offset ``c ∈ \\mathbb{R}^n``, a generator matrix ``G ∈ \\mathbb{R}^{n \times h}`` and an exponent matrix
+``E ∈ \\mathbb{N}^{q×h}_{≥0}``, a sparse polynomial zonotope ``\\mathcal{PZ} ⊂ \\mathbb{R}^n``.
+
+### Fields
+
+- `c` -- constant offset vector
+- `G` -- generators matrix
+- `E` -- exponents matrix
+
+### Notes
+
+Sparse polynomial zonotopes were introduced in N. Kochdumper and M. Althoff.
+*Sparse Polynomial Zonotopes: A Novel Set Representation for Reachability Analysis*. Transactions on Automatic Control, 2021.
+"""
+struct SimpleSparsePolynomialZonotope{N, VN<:AbstractVector{N}, MN<:AbstractMatrix{N}, ME<:AbstractMatrix{<:Integer}}
+    c::VN
+    G::MN
+    E::ME
+
+    function SimpleSparsePolynomialZonotope(c::VN, G::MN, E::ME) where {N<:Real, VN<:AbstractVector{N}, MN<:AbstractMatrix{N}, ME<:AbstractMatrix{<:Integer}}
+        length(c) == size(G, 1) || throw(DimensionMismatch("c and G should have the same number of rows"))
+        size(G, 2) == size(E, 2) || throw(DimensionMismatch("G and E should have the same number of columns"))
+        all(>=(0), E) || throw(ArgumentError("E should have non-negative integers"))
+
+        return new{N, VN, MN, ME}(c, G, E)
+    end
+end
+
+const SSPZ = SimpleSparsePolynomialZonotope
+
+dim(p::SSPZ) = size(p.c, 1)
+ngens(p::SSPZ) = size(p.G, 2)
+nparams(p::SSPZ) = size(p.E, 1)
+order(p::SSPZ) = ngens(p) // dim(p)
+
+center(p::SSPZ) = p.c
+genmat(p::SSPZ) = p.G
+expmat(p::SSPZ) = p.E
+
+"""
+    overapproximate(p::SSPZ, ::Type{Zonotope}; nsdiv=1)
```

Comment:
These `overapproximate` methods can be moved to `Approximations/overapproximate.jl`.